client/landing/gutenboarding/onboarding-block/acquire-intent/site-title.tsx

```diff
@@ -35,13 +35,32 @@ const SiteTitle: React.FunctionComponent< Props > = ( { skipButton, onSubmit } )
 	// translators: label for site title input in Gutenboarding
 	const inputLabel = __( 'My site is called' );
 
+	const placeHolder = useTyper(
+		[
+			'In Focus Photographers',
+			'Cortado Coffee SF',
+			'Leah Rand',
+			'Ava Young',
+			'Mighty Leaf Tea Room',
+		],
+		! siteTitle,
+		{
+			delayBetweenCharacters: 70,
+		}
+	);
+
 	return (
 		<form className="site-title" onSubmit={ handleFormSubmit }>
 			<label htmlFor="site-title__input" className="site-title__input-label">
 				{ inputLabel }
 			</label>
 			<div className="site-title__input-wrapper">
+				{ /* Adding key makes it more performant
+					because without it the element is recreated
+					for every letter in the typing animation
+					*/ }
```

Comment:
Would `React.memo()` have similar effect? If `key` works, that does seem more simple though. 🙃

Reply:
I think they would be the same. But as you said key might be simpler.